crossterm_cursor/src/cursor/ansi.rs

```diff
@@ -6,47 +6,47 @@ use crossterm_utils::{csi, write_cout, Result};
 
 use crate::sys::{get_cursor_position, show_cursor};
 
-use super::ITerminalCursor;
+use super::Cursor;
 
-pub fn get_goto_ansi(x: u16, y: u16) -> String {
+pub(crate) fn get_goto_code(x: u16, y: u16) -> String {
     format!(csi!("{};{}H"), y + 1, x + 1)
 }
 
-pub fn get_move_up_ansi(count: u16) -> String {
+pub(crate) fn get_move_up_code(count: u16) -> String {
     format!(csi!("{}A"), count)
 }
 
-pub fn get_move_right_ansi(count: u16) -> String {
+pub(crate) fn get_move_right_code(count: u16) -> String {
     format!(csi!("{}C"), count)
 }
 
-pub fn get_move_down_ansi(count: u16) -> String {
+pub(crate) fn get_move_down_code(count: u16) -> String {
     format!(csi!("{}B"), count)
 }
 
-pub fn get_move_left_ansi(count: u16) -> String {
+pub(crate) fn get_move_left_code(count: u16) -> String {
     format!(csi!("{}D"), count)
 }
 
-pub static SAVE_POS_ANSI: &'static str = csi!("s");
-pub static RESTORE_POS_ANSI: &'static str = csi!("u");
-pub static HIDE_ANSI: &'static str = csi!("?25l");
-pub static SHOW_ANSI: &'static str = csi!("?25h");
-pub static BLINK_ON_ANSI: &'static str = csi!("?12h");
-pub static BLINK_OFF_ANSI: &'static str = csi!("?12l");
+pub(crate) static SAVE_POS_CODE: &'static str = csi!("s");
```

Comment:
I am a bit unsure about this change, like what do we mean with 'code'? I would like to call it or `ansi_code` or `code`. What do you think of that?

Reply:
What about now? Pushed a change. They end with `_CSI_SEQUENCE` & `_csi_sequence`.

Reply:
Yea good alternatative